seqr/views/apis/anvil_workspace_api.py

```diff
@@ -173,30 +187,44 @@ def _wait_for_service_account_access(user, namespace, name):
     raise TerraAPIException('Failed to grant seqr service account access to the workspace', 400)
 
 
-def _send_load_data_email(project, updated_individuals, data_path, sample_type, user):
-    email_content = """
-        {user} requested to load {sample_type} data ({genome_version}) from AnVIL workspace "{namespace}/{name}" at 
-        "{path}" to seqr project <a href="{base_url}project/{guid}/project_page">{project_name}</a> (guid: {guid})
+def _get_loading_project_path(project, sample_type):
+    return 'gs://seqr-datasets/v02/{genome_version}/AnVIL_{sample_type}/{guid}/'.format(
+        guid=project.guid,
+        sample_type=sample_type,
+        genome_version=GENOME_VERSION_LOOKUP.get(project.genome_version),
+    )
+
 
-        The sample IDs to load are attached.    
+def _send_load_data_slack_msg(project, ids_path, data_path, sample_type, user):
+    pipeline_dag = {
+        "active_projects": [project.guid],
+        "vcf_path": data_path,
+        "project_path": _get_loading_project_path(project, sample_type) + 'v1',
+        "projects_to_run": [project.guid],
+    }
+    message_content = """
+        *{user}* requested to load {sample_type} data ({genome_version}) from AnVIL workspace *{namespace}/{name}* at 
+        {path} to seqr project <{base_url}project/{guid}/project_page|*{project_name}*> (guid: {guid})  
+  
+        The sample IDs to load have been uploaded to {ids_path}.  
+  
+        DAG for the loading pipeline:
+        ```{dag}```
         """.format(
         user=user.email,
         path=data_path,
+        ids_path=ids_path,
         namespace=project.workspace_namespace,
         name=project.workspace_name,
         base_url=BASE_URL,
         guid=project.guid,
         project_name=project.name,
         sample_type=sample_type,
         genome_version=GENOME_VERSION_LOOKUP.get(project.genome_version),
+        dag=json.dumps(pipeline_dag, indent=4),
     )
 
-    send_html_email(
-        email_content,
-        subject='AnVIL data loading request',
-        to=sorted([dm.email for dm in User.objects.filter(is_staff=True, is_active=True)]),
-        attachments=[(
-            '{}_sample_ids.tsv'.format(project.guid),
-            '\n'.join([individual.individual_id for individual in updated_individuals])
-        )]
-    )
+    try:
+        safe_post_to_slack(SEQR_SLACK_ANVIL_DATA_LOADING_CHANNEL, message_content)
```

Comment:
`safe_post_to_slack` is already wrapped in a try-except so we don't need one here